tutorials/messengers/telegram/6_conditions_extras.py

```diff
@@ -59,46 +55,68 @@
 """
 
 
+# %%
+def check_if_latest_message_is_new_chat_member(
+    ctx: Context, _: Pipeline
+) -> bool:
+    if ctx.last_request is None:
+        return False
+    if ctx.last_request.original_message is None:
+        return False
+    if ctx.last_request.original_message.message is None:
+        return False
+    return (
```

Comment:
I feel like it's better to use `try: except AttributeError` here.

Seems very unlikely to me that the conditions above would ever trigger.